### DISCUSSION TITLE
Ramp output frequency using `NFRAMES`

### DISCUSSION THREAD

**Discussion by r-broek** [firemodels/fds#13023] [Q&A] [answered]
Hi!

Is it possible to use a `RAMP` to adjust the value of `NFRAMES` to control when FDS should start outputting data? If so, how would that work? I have read that the frequency of particular output files can be adjusted, however, I would rather not do that for every output file class.

For context, I would like to run a simulation with `T_BEGIN=-10` and `T_END=10` but only want data for $T\geq 0$.

**Comment by mcgratta** [comment, marked as answer]
No, there is not a way to do what you want. However, you can create a single set of `RAMP` lines and have all the relevant `RAMP_XXX` parameters on the `DUMP` line point to it.

**Reply by r-broek** [reply]
Thank you!

**Reply by r-broek** [reply]
> No, there is not a way to do what you want. However, you can create a single set of `RAMP` lines and have all the relevant `RAMP_XXX` parameters on the `DUMP` line point to it.

If I understand correctly, `RAMP_XXXX` specifies discrete timestamps for data output. Let's say I want to output data every 0.5 seconds, does that mean I need to manually input $T=0, T=0.5, \...\ T=9.5, T=10$? Or is there a way to give a timestamp range?

**Reply by mcgratta** [reply]
Yes, you would have to do that.

**Reply by r-broek** [reply]
Thanks again for your response.
